Gemfile

```diff
@@ -87,9 +87,10 @@ group :demo, :development, :test do
 end
 
 group :demo, :development, :heroku, :staging, :production do
```

Comment:
isn't this all our environments? do we need this grouping for redis or can it go into the 'regular' list of gems?

Reply:
all except test...but yes, i guess it's fine if we install it for test env like we do with most of our gems since there's no good reason to exclude it 🤔